clang/lib/AST/ASTContext.cpp

```diff
@@ -3707,6 +3707,21 @@ QualType ASTContext::mergeBoundsSafetyPointerTypes(
   if (OrigDstTy.isNull())
     OrigDstTy = DstTy;
 
+  // An ugly way to keep va_list typedef in DstTy if the merge type doesn't
```

Comment:
@patrykstefanski we don't have to revert this workaround? I guess it's been sticking around for a while?